server/schedule/operator_controller.go

```diff
@@ -409,6 +410,14 @@ func (oc *OperatorController) checkAddOperator(ops ...*operator.Operator) bool {
 			operatorWaitCounter.WithLabelValues(op.Desc(), "exceed-max").Inc()
 			return false
 		}
+		if cl, ok := oc.cluster.(interface{ GetRegionLabeler() *labeler.RegionLabeler }); ok {
```

Comment:
Maybe better to allow operator generated by manual split and scatter request to bypass this check?

Reply:
done